modules/local/xml_vcf/resources/usr/bin/shortvariant.py

```diff
@@ -63,18 +65,101 @@ def extract_data_from_short_variant(short_variant):
         # Extract useful information from the xml file
         'AF': short_variant.get('allele-fraction'),
         'ALT': short_variant.get('alternate-sequence'),
-        '#CHROM': short_variant.get('chromosome'),
+        '#CHROM': short_variant.get('chromosome') if short_variant.get('chromosome') else short_variant.get('position').split(":")[0],
         'DP': short_variant.get('depth'),
-        'POS': short_variant.get('genomic-start'),
+        'POS': short_variant.get('genomic-start') if short_variant.get('genomic-start') else short_variant.get('position').split(":")[-1],
         'REF': short_variant.get('reference-sequence'),
         'SOMATIC-or-GERMLINE': short_variant.get('germline-status'),
         'ZYGOSITY': short_variant.get('tumor-zygosity'),
         'status': short_variant.get('status'),
         'equivocal': short_variant.get('equivocal'),
-        'analytical-only': short_variant.get('analytical-only'),
-        'variant-type': short_variant.get('variant-type')
+        'analytical-only': short_variant.get('analytical-only') if short_variant.get('analytical-only') else "false",
```

Comment:
The value of `analytical-only` is `true` or `false`, which were not correctly retrieved when the values are `false`.

Reply:
NVM, the behaviour was correct.